src/game_inventory.cpp

```diff
@@ -23,8 +23,17 @@ typedef std::function<bool( const item_location & )> item_location_filter;
 namespace
 {
 
-std::string good_bad_none( int value )
+std::string good_bad_none( int value, bool warning = false )
 {
+    if( warning ) {
+        if( value > 0 ) {
+            return string_format( "<color_c_yellow_green>+%d</color>", value );
```

Comment:
I believe `<color_c_yellow_green>` can drop `c_` prefix and be simplified to `<color_yellow_green>`.

Reply:
Yes, you were right. Done and done.